### ISSUE TITLE
Project with your project

### ISSUE THREAD

**Issue by Xenomes** [jasonacox/tinytuya#227]
Hi Jasonacox,

I have made a project to control Tuya in Domoticz with your great plugin at the moment only using cloud but in the future local control. Maybe you like it to mention it on your readme.md. 
Project at https://github.com/Xenomes/Domoticz-TinyTUYA-Plugin

Greetings Xenomes

**Comment by jasonacox** [comment]
Hi @Xenomes !  Nice work!

Are you planning on submitting your plugin as a PR to the [Domoticz](https://github.com/domoticz/domoticz) project?

**Comment by Xenomes** [comment]
No, the plugin will remain separate on github for users to install.

**Comment by jasonacox** [comment]
Thank you for reporting this issue! This issue has been open for over 2 years with no recent activity. We are closing it to help keep the issue tracker focused and manageable.

If this problem is still present in the latest version of TinyTuya, please feel free to reopen this issue or open a new one with updated details (TinyTuya version, Python version, device type, and steps to reproduce). We'd love to help!

Thank you for your contribution to TinyTuya! 🙏
